DSCResourceCommonTests/DSCResourceCommonTests.psd1

```diff
@@ -0,0 +1,58 @@
+@{
+# Version number of this module.
+ModuleVersion = '1.10.0.0'
+
+# ID used to uniquely identify this module
+GUID = 'b3239f27-d7d3-4ae6-a5d2-d9a1c97d6ae4'
```

Comment:
GUID need to be unique, use New-ModuleManifest to generate separate manifests for both modules, then fill in the details.